physics/ephemeris.hpp

```diff
@@ -54,6 +55,11 @@ class Ephemeris {
   using IntrinsicAcceleration =
       std::function<Vector<Acceleration, Frame>(Instant const& time)>;
```

Comment:
Do we still need the old intrinsic acceleration somewhere?  If not, mention that we need to clean up in the issue.

Reply:
We still use it for the non-guided path, which uses a non-RKNG RKN.